module-1/src/Components/Search/Search.tsx

```diff
@@ -1,41 +1,51 @@
 import { useState } from "react";
 
-interface ISearchProps{
-    initialSearchQuery:string, 
-    onSearch:(str:string)=>void
+interface ISearchProps {
+  /**
+   * This is the initial Search Query.
```

Comment:
Here and throughout the project: no need for annotation. It's not bad by any means, just unnecessary in this case.

Reply:
ok  thanks for clarify, actually I put those because I was trying to make storybook to take those comments for documentation, for some reason it worked only once, we will discuss about that